ffms2-sys/build.rs

```diff
@@ -32,9 +33,30 @@ fn main() {
             }
         }).expect("FFMS_LIB_DIR is not set or the specified directory is not valid.");
 
-        // Add the flags for cargo explicitely otherwise added by pkg-config-rs
-        println!("cargo:rustc-link-lib=ffms2");
-        println!("cargo:rustc-link-search={}", lib_dir.to_string_lossy());
+        // Using dynamic library in Windows remains a problem. We have to copy the DLL into a path...
+        // Problem: If 'FFMS_LIB_DIR' is outside of 'target', it is not considered (https://doc.rust-lang.org/cargo/reference/environment-variables.html).
+        #[cfg(windows)] {
+            let mut output = PathBuf::from(env::var("OUT_DIR").expect("Unable to get OUT_DIR"));
+            output.pop(); // We are now in target/debug/build/ffms2-sys-...
+            output.pop(); // We are now in target/debug/build
+            output.pop(); // We are now in target/debug --> This is included in the PATH: https://doc.rust-lang.org/cargo/reference/environment-variables.html#dynamic-library-paths
```

Comment:
I don't like this series of `pop`. Is it possible to retrieve the output path using a specific function of `Path` or find a better way?

Reply:
I intended its usage for documentation purposes with the comments behind. No need for that?

Reply:
Yep, I like the documentation, but I was thinking something like: 

```rust
// We are now in target/debug --> This is included in the PATH: https://doc.rust-lang.org/cargo/reference/environment-variables.html#dynamic-library-paths
let output = Path::new(env::var("OUT_DIR").expect("Unable to get OUT_DIR")).components().take(2).collect();
```

Reply:
Or what's about 
`let output: PathBuf = [env::var("OUT_DIR").expect("Unable to get OUT_DIR").as_ref(), "..", "..", "..", "ffms2.dll"].iter().collect();`
?

So many possibilities :)